packages/ui/src/theme/css/component/textArea.scss

```diff
@@ -1,3 +1,5 @@
+@import 'fieldControl';
```

Comment:
I think `@import` will make a file globally accessible. I think `@use` would be the right way to go